### PR TITLE
[Numpy] FFI: tril_indices

### PR DESCRIPTION
**Pull request by XIAO-XIA**
## Description ##
Use new ffi for tril_indices to get better speed.
||Old FFI (ctypes) (us)|New FFI (cython) (us)|
|:-----|:-----|:-----|
|tril_indices|37.9|18.3|

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by XIAO-XIA** [comment]
@mxnet-bot run ci [unix-cpu, unix-gpu]

**Comment by leezu** [comment]
@XIAO-XIA CI fails in a test related to the Op you're modifying. It suggests there's a bug in your PR

**Comment by XIAO-XIA** [comment]
> @XIAO-XIA CI fails in a test related to the Op you're modifying. It suggests there's a bug in your PR

Thank you very much! I'm trying to fix the bug.

**Comment by XIAO-XIA** [comment]
@mxnet-bot run ci [unix-cpu, unix-gpu]

**Comment by XIAO-XIA** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by XIAO-XIA** [comment]
@mxnet-bot run ci [unix-cpu]

File: src/api/operator/numpy/np_matrix_op.cc [hunk 1]
```diff
@@ -532,4 +532,28 @@ MXNET_REGISTER_API("_npi.squeeze")
...
+  attrs.parsed = param;
+  attrs.op = op;
+  SetAttrDict<op::NumpyTrilindicesParam>(&attrs);
+
+  int num_outputs = 0;
+  auto ndoutputs = Invoke(op, &attrs, 0, nullptr, &num_outputs, nullptr);
+  std::vector<NDArrayHandle> ndarray_handles;
+  ndarray_handles.reserve(num_outputs);
+  for (int i = 0; i < num_outputs; ++i) {
+  ndarray_handles.emplace_back(ndoutputs[i]);
```
Comment (hzfan) [inline src/api/operator/numpy/np_matrix_op.cc:554]:
```suggestion
    ndarray_handles.emplace_back(ndoutputs[i]);
```

**Comment by hzfan** [comment]
CI's green. @yzhliu

**Comment by yzhliu** [comment]
Thanks @XIAO-XIA @hzfan

**Pull request merged**